scrapper.py

```diff
@@ -48,14 +56,30 @@ def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
```

Comment:
this will work
however, is there a way to implement this without try/except tricks?